istio/cache/db.go

```
@@ -0,0 +1,66 @@
+package cache
+
+import (
+	"sync"
+	"time"
+
+	"github.com/stackrox/istio-cves/types"
+	"github.com/stackrox/scanner/pkg/istioUtil"
+	"github.com/stackrox/scanner/pkg/vulndump"
+)
+
+var (
+	_ Cache = (*cacheImpl)(nil)
+)
+
+type cacheImpl struct {
+	cacheRWLock sync.RWMutex
+
+	cache map[string]types.Vuln
+
+	dir             string
+	timeRWLock      sync.RWMutex
+	lastUpdatedTime time.Time
+}
+
+func (c *cacheImpl) GetVulnsByVersion(version string) []types.Vuln {
+	c.cacheRWLock.RLock()
+	defer c.cacheRWLock.RUnlock()
+
+	var vulns []types.Vuln
+	for _, vuln := range c.cache {
+		isAffected, _, _ := istioUtil.IstioIsAffected(version, vuln)
```

can you add a comment why we ignore errors here?